presto/tests/docker/docker-compose.yaml

```diff
@@ -0,0 +1,45 @@
+version: '3'
+
+# Adapted from https://github.com/Lewuathe/docker-presto-cluster
+services:
+  coordinator:
+    build:
+      context: ./config
+      args:
+        NODE_TYPE: coordinator
+        NODE_ID: coordinator
+        VERSION: latest
+    container_name: presto_coordinator
+    # labels:
```

Comment:
Do we need these commented lines?

Reply:
I changed it to PY37, but we need something in order for the E2E environment to be able to spin up 🙂